src/lib/config.ts

```diff
@@ -243,6 +243,9 @@ export const HIDE_UNVERSIONED_ENDPOINT = boolVar(
 export const METRICS_MAX_REPORT_INTERVAL_SECONDS = Math.round(
 	intVar('METRICS_MAX_REPORT_INTERVAL', 1 * MINUTES) / 1000,
 );
+export const DOWNLOAD_PROGRESS_MAX_REPORT_INTERVAL_SECONDS = Math.round(
+	intVar('DOWNLOAD_PROGRESS_MAX_REPORT_INTERVAL_SECONDS', 30) / 1000,
```

Comment:
So the default value is 30ms?
Should this be:
```
intVar('DOWNLOAD_PROGRESS_MAX_REPORT_INTERVAL', 30) / 1000,
```
or
```
intVar('DOWNLOAD_PROGRESS_MAX_REPORT_INTERVAL_SECONDS', 30 * SECONDS) / 1000,
```

Reply:
Oh I see that https://github.com/balena-io/open-balena-api/pull/887 fixed that :+1: